src/Coord.py

```diff
@@ -0,0 +1,16 @@
+from chess_enums import Space
+
+
+class Coord:
+    def __init__(self, c: int, r: int) -> None:
+        self.c = c
+        self.r = r
+
+    @classmethod
+    def fromSpace(cls, par_space: Space) -> None:
+        c = par_space.value / 8
```

Comment:
We can remove all instances of ".value" for the enums now that they are IntEnums.

Reply:
Good point, removed